### PR TITLE
Added warning on using old engine versions in launch tab

### PR DESCRIPTION
**Pull request by yaustar**
image

### CODE REVIEW

File: content/en/user-manual/scripting/custom_engine.md [hunk 1]
```diff
@@ -48,6 +48,11 @@ To launch with the minified build, use:
 
     https://launch.playcanvas.com/<scene_id>?use_local_engine=https://code.playcanvas.com/playcanvas-0.225.0.min.js
 
+<div class="alert alert-info">
+    The Editor only officially supports the current Engine release and the previous minor version. While it is sometimes possible to use an older version of the Engine, it is not a long-term supported workflow. We strongly recommend keeping live projects updated with the current Engine release.
+</div>
+
+
```
Comment (willeastcott) [inline content/en/user-manual/scripting/custom_engine.md:55]: Only put a single line between blocks of Markdown text. I should create a style guide. But you'll see it's like that throughout the entire content base.

File: content/en/user-manual/version-control/index.md [hunk 2]
```diff
@@ -26,6 +26,24 @@ Merging works in a similar way to other VCSs. However, in PlayCanvas it's import
...
 
+## Storage Usage with Version Control
+
+Using Version Control will use more storage for checkpoints and branches.
+
+Every checkpoint created only stores the changes from the previous checkpoint. The amount of data used is dependent on the changes such as new assets, texture changes etc.
+
+Creating a branch from a checkpoint makes a copy of the project state at that checkpoint. This can increase storage usage significantly depending on the project.
+
+Unfortunately, it is not possible to delete branches or checkpoints expect in these specific cases:
```
Comment (willeastcott) [inline content/en/user-manual/version-control/index.md:37]:
```suggestion
Unfortunately, it is not possible to delete branches or checkpoints except in these specific cases:
```

File: content/en/user-manual/scripting/custom_engine.md [hunk 3]
```diff
@@ -48,6 +48,11 @@ To launch with the minified build, use:
 
     https://launch.playcanvas.com/<scene_id>?use_local_engine=https://code.playcanvas.com/playcanvas-0.225.0.min.js
 
+<div class="alert alert-info">
+    The Editor only officially supports the current Engine release and the previous minor version. While it is sometimes possible to use an older version of the Engine, it is not a long-term supported workflow. We strongly recommend keeping live projects updated with the current Engine release.
+</div>
+
```
Comment (yaustar) [inline content/en/user-manual/scripting/custom_engine.md:54]:
```suggestion
```

File: content/en/user-manual/version-control/index.md [hunk 4]
```diff
@@ -26,6 +26,24 @@ Merging works in a similar way to other VCSs. However, in PlayCanvas it's import
 
 More details can be found on the specific pages for [checkpoints][1], [branches][2], and [merging][3].
 
+## Storage Usage with Version Control
```
Comment (willeastcott) [inline content/en/user-manual/version-control/index.md:29]: ## How Version Control Affects Your Storage

**Review by willeastcott — APPROVED** [review]
Approving with minor comments

**Pull request merged**